shared/web_features.go

```diff
@@ -0,0 +1,109 @@
+// Copyright 2024 The WPT Dashboard Project. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package shared
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"strings"
+)
+
+// WebFeaturesData is the public data type that represents the data parsed from
+// a given manifest file
+type WebFeaturesData map[string]map[string]interface{}
+
+// ErrUnknownWebFeaturesManifestVersion indicates that the parser does not know how to parse
+// this version of the web features file.
+var ErrUnknownWebFeaturesManifestVersion = errors.New("unknown web features manifest version")
```

Comment:
These Err* variables can be private, as it won't be used outside of the packages?

Reply:
This is true. We can make them private.